File: src/main/java/net/eduard/permissions/command/PermissionsHelpCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class PermissionsHelpCommand extends CommandManager {

	public PermissionsHelpCommand() {
		super("help", "ajuda", "?");
		setDescription("Ver como usa os comandos");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		for (CommandManager sub : getParent().getCommands().values()) {
			if (sender.hasPermission(sub.getPermission())) {
				sender.sendMessage("§3"+sub.getUsage() + " §b" + sub.getDescription());
			}
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsHelpCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class PermissionsHelpCommand : CommandManager("help", "ajuda", "?") {
    override fun onCommand(
        sender: CommandSender,
        command: Command,
        label: String,
        args: Array<String>
    ): Boolean {
        for (sub in parent!!.getCommands().values) {
            if (sender.hasPermission(sub.permission)) {
                sender.sendMessage("§3" + sub.usage + " §b" + sub.description)
            }
        }
        return true
    }

    init {
        description = "Ver como usa os comandos"
    }
}